cupy/cuda/compiler.py

```diff
@@ -250,7 +257,9 @@ def _preprocess(source, options, arch, backend):
 
         prog = _NVRTCProgram(source, '')
         try:
+
             result, _ = prog.compile(options)
+
```

Comment:
Just a nitpick: could you remove the empty lines?

Reply:
Fixed.